checks/raw/pinned_dependencies_test.go

```diff
@@ -65,6 +65,11 @@ func TestGithubWorkflowPinning(t *testing.T) {
 			name:     "Matrix as expression",
 			filename: "./testdata/.github/workflows/github-workflow-matrix-expression.yaml",
 		},
+		{
+			name:     "Can't detect OS, but still detects unpinned Actions",
+			filename: "./testdata/.github/workflows/github-workflow-unknown-os.yaml",
+			warns:    2, // 1 in job with unknown OS, 1 in job with known OS
```

Comment:
do these warn numbers (or the comments) needed updated since the statement is an info?